app/build.gradle

```diff
@@ -99,6 +97,7 @@ android {
 
             buildConfigField("boolean", "CRASHLYTICS_ENABLED", "true")
             buildConfigField("boolean", "ANALYTICS_ENABLED", "true")
+            buildConfigField("boolean", "IS_PROD", "true")
```

Comment:
But we can use `!BuildConfig.DEBUG` to determine if it's not release build, right? Isn't it redundant?

Reply:
I would name it as "isDevNetEnabled" and that's all.
BuildConfig.DEBUG can be responsible for a lot of things, so it's better to separate flags for different things and use them.